newsroom/web/default_settings.py

```diff
@@ -406,7 +406,7 @@
     },
     "newsroom:send_scheduled_notifications": {
         "task": "newsroom.notifications.send_scheduled_notifications.send_scheduled_notifications",
-        "schedule": crontab(minute="*/1"),
+        "schedule": crontab(minute="*/5"),
```

Comment:
@petrjasek Setting this to run every minute didn't make sense, as you pointed out we can't provide a guarantee that it would run on such a granular level.

It also raises another concern/confusion, that we're providing the user an input that allows any time to be placed in. Ideally we should provide something like 15min slots they can choose from (we can discuss this as an improvement to the UX/system behaviour).